apps/src/applab/appStorage.js

```diff
@@ -33,6 +33,10 @@ var handleGetKeyValue = function(onSuccess, onError) {
     onSuccess(undefined);
     return;
   }
+  if (this.status === 429) {
```

Comment:
onErrorStatus helper added in follow-up PR https://github.com/code-dot-org/code-dot-org/pull/7098